create-custom-map-style/README.md

```diff
@@ -0,0 +1,990 @@
+
+	```
+	2021-00-00X00:00:00.000000+00:00  arn:aws:geo:us-west-2:{accountNumber}:map/CreateCustomMapStyle CreateCustomMapStyle
+	```
+
+2. Navigate to [AWS Location Services Console](console.aws.amazon.com/location/home) and check under Maps, if the Map resource have been created. 
+
+3. We are using Esri Street Map as our base map style. This comprehensive street map includes highways, major roads, minor roads, railways, water features, cities, parks, landmarks, building footprints, and administrative boundaries. 
+
+4. Set IAM Permissions
+
+	```
+	./setIamPermissions.sh <YOUR COGNITO AUTH ROLE NAME>  Amplify-Default <YOUR ACCOUNT ID> us-west-2
+	```
+
+5. Navigate to [AWS IAM Console](aws.amazon.com/iam/home) , check under Roles if the role you just created in the above step exists, click on the Role Name > Policy Name and you will be able to find the Policy (JSON object) which will look like this :
+
+```
+
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "PlaceIndexReadOnly",
+            "Effect": "Allow",
+            "Action": [
+                "geo:GetMapStyleDescriptor",
+                "geo:GetMapGlyphs",
+                "geo:GetMapSprites",
+                "geo:GetMapStyleDescriptor",
+                "geo:GetMapTile"
+            ],
+            "Resource": "arn:aws:geo:<YOUR ACCOUNT ID>:us-west-2:map/CreateCustomMapStyle"
+        },
+        {
+            "Sid": "MapsReadOnly",
+            "Effect": "Allow",
+            "Action": [
+                "geo:GetMapStyleDescriptor",
+                "geo:GetMapGlyphs",
+                "geo:GetMapSprites",
+                "geo:GetMapStyleDescriptor",
+                "geo:GetMapTile"
+            ],
+            "Resource": "arn:aws:geo:<YOUR ACCOUNT ID>:us-west-2:map/CreateCustomMapStyle"
+        }
+    ]
+}
+
+```
+
+### React - Amazon Location Service Integration
+
+1. Now it's time for adding yet another package; this time a very important library that uses WebGL to render interactive maps from [vector tiles](https://docs.mapbox.com/help/glossary/vector-tiles/) and [Mapbox styles](https://docs.mapbox.com/mapbox-gl-js/style-spec/)
+
+	```
+	npm install mapbox-gl --save
+	```
+
+2. With this we'll start adding the map to our page. Start by importing the dependencies in your `SamplePage` component
+
+	```
+	...
+	import  MapboxWorker  from  'worker-loader!mapbox-gl/dist/mapbox-gl-csp-worker';
+	import  mapboxgl  from  'mapbox-gl';
+	...
+	```
+
+3. Next fill out some boilerplate
+
+	```
+	...
+	// Below your import statements
+	mapboxgl.workerClass = MapboxWorker;
+	mapboxgl.accessToken = 'xx.xxX0XxxxXXxxxXXxXXXxXXXxXxxxX0x0XX01XXXxXX00xxXxx0X0x000x0X0XXX0.xX0X0XXX0x0XxXxXxx0xXx';
+	// This is the default Mapbox access token
+	...
+	```
+
+4.  Unfortunately Mapbox requires an access token to even instantiate their MapboxGL class, so be sure generate your own one [here](https://account.mapbox.com/access-tokens).
```

Comment:
Switching to MapLibre will allow you to get rid of this. Please do.

Reply:
Just switched the React code to use Maplibre